src/components/UserSearchBar.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Button, TextField, Typography, Link, InputBase, Paper, createStyles, Theme, withStyles } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+interface SearchProps {
+  onSearchChange: any;
+  classes: { search: string; searchWrapper: string; inputWrapper: string; };
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    search: {
+      color: '#828282',
```

Comment:
The search icon looked like it was a little offcenter from the search. To regulate you can restrict the dimensions to be proportional to searchWrapper. I put a `height: 75%` and it looks nicer. You can play around and see if there's anything better but feel free to use 75% otherwise.

Current:
![image](https://user-images.githubusercontent.com/35501399/104109856-d6eec680-5286-11eb-81ee-ffa6ef377180.png)

Changed icon dimension:
![image](https://user-images.githubusercontent.com/35501399/104109830-8f683a80-5286-11eb-88da-3d409a6d2364.png)